src/meta/processors/admin/CreateSnapshotProcessor.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_CREATESNAPSHOTPROCESSOR_H_
+#define META_CREATESNAPSHOTPROCESSOR_H_
+
+#include <gtest/gtest_prod.h>
+#include "meta/processors/BaseProcessor.h"
+
+namespace nebula {
+namespace meta {
+
+class CreateSnapshotProcessor : public BaseProcessor<cpp2::ExecResp> {
+public:
+    static CreateSnapshotProcessor* instance(kvstore::KVStore* kvstore) {
+        return new CreateSnapshotProcessor(kvstore);
+    }
+    void process(const cpp2::CreateSnapshotReq& req);
+
+    std::string genSnapshotName();
```

Comment:
mark this method to private would be better ?

Reply:
excellent